src/MCPClient/lib/clientScripts/getAipStorageLocations.py

```diff
@@ -20,6 +20,7 @@ def get_aip_storage_locations(purpose):
     choices = {}
     for storage_dir in storage_directories:
         choices[storage_dir['description']] = storage_dir['resource_uri']
+    choices['Default location'] = '/api/v2/location/default/{}/'.format(purpose)
```

Comment:
The automated workflow chooses `/api/v2/location/default/AS/` but MCPServer won't recognize it as a valid option unless it's previously listed by `getAipStorageLocations.py`, that's the reason behind this change.